lib/host/wasi/wasifunc.cpp

```diff
@@ -1110,6 +1224,7 @@ Expect<uint32_t> WasiFdWrite::body(const Runtime::CallingFrame &Frame,
 Expect<uint32_t>
 WasiPathCreateDirectory::body(const Runtime::CallingFrame &Frame, int32_t Fd,
                               uint32_t PathPtr, uint32_t PathLen) {
+  // no alignment checks for PathPtr, as char has alignment of 1 byte.
```

Comment:
Please unify the style of your comment (you used "PathBufPtr should be aligned to at least 1 byte (which is always true)" above)